### ISSUE TITLE
[popup] 'Enable Logs' only working for other users

### ISSUE THREAD

**Issue by MrEous** [tprouvot/Salesforce-Inspector-reloaded#635]
**Make sure to read the troubleshooting section before creating an issue**

**Describe the bug**
When logged into SF normally (as yourself) the 'Enable Logs' button is not initiating the trace log.

**To Reproduce**
1. Open Inspector
2. Click on Users tab
3. Click 'Enable Logs'
4. Check Setup>Debug Logs and verify there are no new logs recorded

**Expected behavior**
Enable Logs should initiate the trace log for self-user

**Desktop (please complete the following information):**
 - OS: Win11
 - Browser: Chrome
 - Version: 1.24

**Comment by tprouvot** [comment]
The default loglevel name configured in the extension's setting is the default one from the dev console. It may not be created in your org, so you would need to create your own one and set the name in the settings or let the default one but open the Salesforce dev console before.

**Comment by tprouvot** [comment]
Let me know if you could solve the issue based on my comments, otherwise I'll re open the issue

**Comment by MrEous** [comment]
"SFDC_DevConsole" is in the Org, and it works for any other users I select in the Users tab of Inspector.  It does not work when attempting to run a debug on myself.

**Comment by tprouvot** [comment]
Could you check if you have an error in the chrome dev console?

**Comment by MrEous** [comment]
I checked Console within Chrome dev console and there are no errors running it as myself.
If you run default SFDC_DevConsole through Enable Logs on yourself it is recording as expected?

**Comment by tprouvot** [comment]
Yes the feature is working properly, could you add a screen recording to show what is the behaviour ?

**Comment by tprouvot** [comment]
I'm closing the issue and will re-open it when more info will be added
